tests/models.py

```diff
@@ -23,14 +23,18 @@ class ShardedModelIDs(TableStrategyModel):
 # the mixin provided in order to save the shard on the user.
 @shard_storage_config()
 class User(AbstractUser, ShardedByMixin):
-    pass
+    # Patch for Django 1.11
+    groups = None
```

Comment:
In Django 1.11 `django.core.management.base.SystemCheckError: SystemCheckError: System check identified some issues:` is raised when run migrations.

```
ERRORS:
tests.PostgresShardUser.groups: (fields.E304) Reverse accessor for 'PostgresShardUser.groups' clashes with reverse accessor for 'User.groups'.
        HINT: Add or change a related_name argument to the definition for 'PostgresShardUser.groups' or 'User.groups'.
tests.PostgresShardUser.user_permissions: (fields.E304) Reverse accessor for 'PostgresShardUser.user_permissions' clashes with reverse accessor for 'User.user_permissions'.
        HINT: Add or change a related_name argument to the definition for 'PostgresShardUser.user_permissions' or 'User.user_permissions'.
tests.User.groups: (fields.E304) Reverse accessor for 'User.groups' clashes with reverse accessor for 'PostgresShardUser.groups'.
        HINT: Add or change a related_name argument to the definition for 'User.groups' or 'PostgresShardUser.groups'.
tests.User.user_permissions: (fields.E304) Reverse accessor for 'User.user_permissions' clashes with reverse accessor for 'PostgresShardUser.user_permissions'.
        HINT: Add or change a related_name argument to the definition for 'User.user_permissions' or 'PostgresShardUser.user_permissions'.
```